sklearn/linear_model/tests/test_ridge.py

```diff
@@ -49,8 +50,11 @@
 X_iris = sp.csr_matrix(iris.data)
 y_iris = iris.target
 
-DENSE_FILTER = lambda X: X
-SPARSE_FILTER = lambda X: sp.csr_matrix(X)
+
+def DENSE_FILTER(X): return X
```

Comment:
Please avoid changes that are not related to your PR. It makes the review less pleasant for everyone involved. Can you put back the lambdas?

Reply:
I'm sorry, I don't know why this keeps happening. I had already reverted those changes.